### ISSUE TITLE
request to add a proxy

### ISSUE THREAD

**Issue by AngeloHH** [N3koSempai/Everling-Browser#8]
Is it possible that a proxy connection can be added in the future?

**Comment by N3koSempai** [comment]
Yes.  that would enter the advanced features stage.  For now, due to the economic situation in my country, I am working unrelated to programming.  but in December I will resume work on this project.  if you want to contribute by writing code feel free to do so.  The code will be reviewed and approved if applicable.
